codebase/superdarn/src.lib/tk/radar.1.22/src/radar.c

```diff
@@ -43,7 +43,9 @@ struct RadarSite *RadarEpochGetSite(struct Radar *ptr,double tval) {
 
   /* If tval is before the radar.dat start time of the radar
      then return NULL */
-  if ((ptr->st_time !=-1) && (tval<ptr->st_time)) return NULL;
+  if ((ptr->st_time !=-1) && (tval<ptr->st_time)) {
+      fprintf(stderr,"Warning: Data is before commissioning date set in the radar.dat file\n");   
```

Comment:
As @ecbland mentioned, this doesn't make much sense for things like `fov_plot`. This if is checking `tval`, so the time that was input, so a better message might reference time.